lighthouse-core/audits/byte-efficiency/offscreen-images.js

```diff
@@ -28,8 +28,8 @@ const UIStrings = {
 
 const str_ = i18n.createMessageInstanceIdFn(__filename, UIStrings);
 
-const ALLOWABLE_OFFSCREEN_X = 100;
-const ALLOWABLE_OFFSCREEN_Y = 200;
+const ALLOWABLE_OFFSCREEN_IN_PX = 100;
+const ALLOWABLE_OFFSCREEN_BOTTOM_IN_VIEWPORTS = 3;
```

Comment:
add a comment pointing to https://github.com/GoogleChrome/lighthouse/issues/10471 for justification